src/olympia/legacy_discovery/views.py

```diff
@@ -103,7 +102,9 @@ def from_api(list_type):
         ctx = {'featured_addons': from_api('featured')}
     elif section == 'up-and-coming':
         ctx = {'up_and_coming': from_api('hotness')}
-    return render(request, 'legacy_discovery/more_addons.html', ctx)
+
+    content = render(request, 'legacy_discovery/more_addons.html', ctx)
+    return content
```

Comment:
Irrelevant, I used this for debugging. I'll remove this and a few other unrelated changes.